cmd/nomad-toast/allocations/command.go

```diff
@@ -22,6 +28,31 @@ func RegisterCommand(rootCmd *cobra.Command) error {
 		},
 	}
 
+	flags := cmd.Flags()
```

Comment:
I would love to add some validation of the valid arguments each flag can take (aka the valid client states defined by the nomad API), but I could not find a robust and easy way to do this with `cobra` out of the box. 

Can you suggest anything?

Reply:
I don't know how to add function into Cobra like that and I don't have time to look it up right now, so I've gone for your minimal suggestion of using `strings.ToLower` to avoid case problems